src/styles/ui-playground/_shared.scss

```diff
@@ -0,0 +1,168 @@
+@use '../reset';
```

Comment:
This is moved from `ui-playground.scss`. Nothing new here except for a `font-size` change to `h2` (it's in its own commit for clarity).